lib/vagrant-openshift/action/run_openshift_tests.rb

```diff
@@ -92,6 +92,12 @@ def call(env)
           cmd = cmd_env.join(' ') + ' ' + build_targets.join(' ')
           env[:test_exit_code] = run_tests(env, [cmd], true)
 
+          if env[:test_exit_code] == 0 && @options[:extended_test_packages].length > 0
+            extended_cmd = 'hack/test-extended.sh ' + Shellwords.escape(@options[:extended_test_packages])
```

Comment:
are we already using `Shellwords` in vagrant-openshift? I didn't see it anywhere else...